static/app/views/performance/data.tsx

```diff
@@ -396,16 +392,6 @@ export function getTermHelp(
   return PERFORMANCE_TERMS[term](organization);
 }
 
-function isUsingLimitedSearch(location: Location, withStaticFilters: boolean) {
-  const {query} = location;
-  const mepSearchState = decodeScalar(query[METRIC_SEARCH_SETTING_PARAM], '');
-  const mepSettingState = decodeScalar(query[METRIC_SETTING_PARAM], ''); // TODO: Can be removed since it's for dev ui only.
-  return (
-    withStaticFilters &&
-    (mepSearchState === MEPState.metricsOnly || mepSettingState === MEPState.metricsOnly)
-  );
-}
-
```

Comment:
not sure if this can be removed, but this will fail when navigating from transaction summary back to performance